test/unit/org/apache/cassandra/db/compaction/CompactionStrategyManagerPendingRepairTest.java

```diff
@@ -388,15 +373,6 @@ public void testFinalizedAndCompactionRace() throws IOException
         Assert.assertSame(RepairFinishedCompactionTask.class, compactionTask.getClass());
         compactionTask.execute();
 
-        System.out.println("*********************************************************************************************");
```

Comment:
Whether we got a task to execute above or not, the checks below (and maybe the printouts) should succeed.

Reply:
Agreed, that's the gist of the test